packages/core/src/civil.ts

```diff
@@ -142,6 +142,16 @@ export class Civil {
     return CivilTCR.singleton(this.ethApi, this.contentProvider);
   }
 
+  /**
+   * Same as `initializeMultisig` but supports (but does not require) a multisig proxy.
+   * @param multisigAddress (optional) Multisig through which to proxy interactions with this TCR
+   * @returns Promise containing singleton instance of TCR living on the current network
+   * @throws {CivilErrors.UnsupportedNetwork} In case we're trying to get a non-deployed singleton
+   */
+  public async tcrSingletonTrustedMultisigSupport(multisigAddress?: EthAddress): Promise<CivilTCR> {
```

Comment:
Maybe a bit weird creating this function, but the existing `tcrSingletonTrusted` function is synchronous and this one needs to be async, and it seemed silly to update the dozens of existing calls to `tcrSingletonTrusted` (and every branch that leads to those calls) to be async too, especially when only like 2 of them actually need multisig.